src/core/layout/navbar/responsive-drawer/drawer/drawer-component.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback } from 'react'
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/128524215-992cc1ec-9298-44bb-a753-7694c5833bee.png)
Let's break up this folder into separate `dektop` and `mobile` folders.

Reply:
Done